apex/contrib/csrc/groupbn/cuda_utils.h

```diff
@@ -8,7 +12,11 @@ namespace cuda {
 namespace utils {
 
 static inline int MaxSharedMemoryPerMultiprocessor(int device_id) {
+#ifdef __HIP_PLATFORM_HCC__
+    return getDeviceProperties(device_id)->maxSharedMemoryPerMultiProcessor;
+#else
     return getDeviceProperties(device_id)->sharedMemPerMultiprocessor;
+#endif
 }
```

Comment:
@micmelesse Do you see any risk in adding this mapping: `sharedMemPerMultiprocessor`->`maxSharedMemoryPerMultiProcessor` to the mappings file? It's not exactly a type or an api name, but it seems like it should be fine to add in the mappings file, even if it was just a variable name?

Reply:
It should be fine.